SampleBase/CMakeLists.txt

```diff
@@ -72,6 +72,11 @@ elseif(PLATFORM_ANDROID)
         include/SampleApp.hpp
     )
 elseif(PLATFORM_LINUX)
+    if(PLATFORM_FREEBSD)
+        if(EXISTS /usr/local/include)
+            include_directories(/usr/local/include)
```

Comment:
Defining global include directory is a discouraged practice that we don't use in Diligent.
If this directory is needed (and why, BTW), it needs to be added to `Diligent-BuildSettings` or `Diligent-PrivateBuildSettings`

Reply:
I'll take care of this(see other pr) once I get home.  It is needed because everything not in base is installed to /usr/local.